packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -43,7 +47,7 @@ export function useTableColumnResize<T>(props: ResizerProps<T>, state: ColumnRes
       }
       if (e.key === 'Escape' || e.key === 'Enter' || e.key === ' ') {
         // switch focus back to the column header on escape
-        const columnHeader = ref.current.previousSibling as HTMLElement;
+        const columnHeader = ref.current.parentElement as HTMLElement;
```

Comment:
Can we actually assume that the parentElement is the column header? This would prevent users from placing any wrapper elements between.